### PR TITLE
Create fetch-version library

### PR DESCRIPTION
**Pull request by wkaluza**

### CODE REVIEW

File: apps/etch/main.cpp [hunk 1]
```diff
@@ -303,7 +302,7 @@ int main(int argc, char **argv)
   }
 
   // print the header
-  fetch::commandline::DisplayCLIHeader("vm-lang");
+  fetch::version::DisplayCLIHeader("vm-lang");
```
Comment (ejfitzgerald) [inline apps/etch/main.cpp]:
```suggestion
  fetch::version::DisplayCLIHeader("etch");
```

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]
Just a minor request

File: apps/etch/main.cpp (hunk 1 above)
Comment (wkaluza) [reply apps/etch/main.cpp]: Good spot!

**Review by ejfitzgerald — APPROVED** [review]
LGTM

**Pull request merged**